benchflow-dsl/src/test/scala/cloud/benchflow/dsl/definition/configuration/goal/explorationspace/explorationvalues/ByteValuesTest.scala

```diff
@@ -0,0 +1,53 @@
+package cloud.benchflow.dsl.definition.configuration.goal.explorationspace.explorationvalues
+
+import net.jcazevedo.moultingyaml._
+import org.junit.{ Assert, Test }
+import org.scalatest.Matchers._
+import org.scalatest.junit.JUnitSuite
+import cloud.benchflow.dsl.definition.configuration.goal.explorationspace.explorationvalues.ExplorationValuesBytesYamlProtocol._
+
+import scala.util.Try
+
+/**
+ * @author Jesper Findahl (jesper.findahl@gmail.com)
+ *         created on 2017-05-23
+ */
+class ByteValuesTest extends JUnitSuite {
+
+  private val valuesYaml: String =
+    """
+      |values: [1m,2m,10m,32m]
+    """.stripMargin
+
+  private val rangeYaml: String =
+    """
+      |range: [1g,10g]
```

Comment:
In this case it is the default integer step, e.g. `+`. To add another step we wanted to do that with a function. I asked Igor about support for that because it involves parsing etc.